src/components/Calculator.js

```diff
@@ -1,14 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import InputButton from './InputNumberBtn';
 import './Calculator.css';
+import calculate from '../logic/calculate';
 
-const number = ['AC', '\u00B1', '%', '\u00F7', 7, 8, 9, '\u00D7', 4, 5, 6, '-', 1, 2, 3, '+', 0, '.', '='];
+const number = ['AC', '\u00B1', '%', '÷', 7, 8, 9, 'x', 4, 5, 6, '-', 1, 2, 3, '+', 0, '.', '='];
 
 function Calculator() {
+  const [result, setResult] = useState({ total: null, next: null, operation: null });
+  const { total, next, operation } = result;
+
+  const handleInputBtn = (e) => {
+    const input = e.target.innerHTML;
+    setResult(calculate(result, input));
+  };
```

Comment:
- [ ] As required for the project all calculator buttons to work according to their functionality, kindly ensure that you check the `+/-` button and fix its functionality to give the expected result once clicked as now it is not the case. Below is error get once clicked the button:
![Screenshot from 2023-07-04 05-41-21](https://github.com/ichsansandy/math-magician-microverse/assets/83042742/fcb9e91f-4837-4f69-9ede-71ebe3896b60)